fdm-app/app/components/custom/atlas-fields-panels.tsx

```
@@ -1,118 +1,139 @@
+import { Check, Info } from "lucide-react"
+import { useEffect, useState } from "react"
+import { data, useFetcher } from "react-router"
+import { LoadingSpinner } from "./loadingspinner"
+
+export function FieldsPanelZoom({
+    zoomLevelFields,
+}: { zoomLevelFields: number }) {
+    const { current: map } = useMap()
+    const [panel, setPanel] = useState(<></>)
 
     useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: zoomLevelFields

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)